### PR TITLE
feat(PMS-165): Create blueprint from database

### PR DESCRIPTION
**Pull request by oHTGo**

### CODE REVIEW

File: integration/db/build.gradle [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    runtimeOnly 'com.microsoft.sqlserver:mssql-jdbc:11.2.3.jre17'
+    runtimeOnly 'com.h2database:h2'
+
+    liquibase 'com.microsoft.sqlserver:mssql-jdbc:11.2.3.jre17'
+}
+
+var driver = "com.microsoft.sqlserver.jdbc.SQLServerDriver"
+var username = "sa"
+var password = "P@ssw0rd000"
+var url = "jdbc:sqlserver://localhost:1433;trustServerCertificate=true;databaseName=F-PMS"
```
Comment (huybui38) [inline integration/db/build.gradle:29]: Please help move this to the property file

File: integration/db/build.gradle [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    classpath sourceSets.main.runtimeClasspath
+    mainClass = "liquibase.integration.commandline.Main"
+
+    args "--username=${username}"
+    args "--password=${password}"
+    args "--url=${url}"
+    args "--driver=${driver}"
+    args "--changeLogFile=src/main/resources/changelogs/changelog-${now()}.yaml"
+    args "--referenceUrl=hibernate:spring:${group}.integration.db" +
+            "?dialect=org.hibernate.dialect.SQLServerDialect"
```
Comment (huybui38) [inline integration/db/build.gradle:65]: Apply physical_naming_strategy here, then we will no longer need to specify the column names.

File: integration/db/src/main/java/com/hbc/pms/integration/db/entities/SensorConfigurationFigure.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+@Table(name = "sensor_configuration_figure")
+@Builder
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class SensorConfigurationFigure extends BaseEntity {
+
+  @Column(name = "name")
+  private String name;
+
+  @Column(name = "data_type")
```
Comment (huybui38) [inline integration/db/src/main/java/com/hbc/pms/integration/db/entities/SensorConfigurationFigure.java:17]: No need to specific the name on every column, use the naming strategy  CamelCaseToUnderscoresNamingStrategy

File: integration/db/src/main/resources/db.yml [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+    properties:
+      hibernate.default_batch_fetch_size: 100
+
+---
+spring.config.activate.on-profile: local
+
+spring:
+  jpa:
+    hibernate:
+      ddl-auto: create
```
Comment (huybui38) [inline integration/db/src/main/resources/db.yml:15]: This may cause conflicts with the Liquibase integrations.

Comment (huybui38) [reply integration/db/src/main/resources/db.yml:15]: Use `spring.liquibase.enabled` to enable Liquibase migration’s execution on startup

File: integration/plc-integration/src/main/resources/mokka7.properties [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+err_2100000=partial data read
+err_2200000=buffer too small
+err_2300000=function refused
+err_2400000=destroying
+err_2500000=invalid param number
+err_2600000=cannot change param
+err_2700000=function not implemented
+
+err_8104=this service is not implemented on the module or a frame error was reported
+err_d401=information function unavailable
```
Comment (huybui38) [inline integration/plc-integration/src/main/resources/mokka7.properties:60]: vc bữa quen copy cai nay

File: .run/CoreApiApplication.run.xml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+<component name="ProjectRunConfigurationManager">
```
Comment (huybui38) [inline .run/CoreApiApplication.run.xml:1]: I guess we forgot to ignore this file, it seems like this is your custom-run profile

**Review by zKirinz — APPROVED** [review]

**Pull request merged**